### ISSUE TITLE
Display something when there are no open tabs

### ISSUE THREAD

**Issue by dj3500** [dj3500/hightail#59]
An empty window with just a menu bar perhaps doesn't look very nice. We could say something like "To add  a problem, select ..."

**Comment by erilyth** [comment]
I thought if we could add some text as you proposed in one of the tabs and name it lets say, "Getting Started" or something similar, would that do ?

**Comment by Einsteino** [comment]
We could build on that, and add buttons for adding a new problem/contest, and more, like in most other software.

**Comment by dj3500** [comment]
I don't think we need buttons. The user can easily figure out what to do once they're in the File menu.
